### PR TITLE
Addition of Via Rail and Train de Charlevoix

### PR DESCRIPTION
**Pull request by davidbellerive**
Using their respective GTFS data, added the following elements to the map:

> Via Rail Canada: alignment, stops (including flag stops), icon, main lines / sectors and maintenance centres.

> Train de Charlevoix: alignment, stops, icon.

### CODE REVIEW

File: src/config/config.json [hunk 1]
```diff
@@ -0,0 +1,209 @@
+{
```
Comment (dellisd) [inline src/config/config.json:1]: This file should be deleted (it looks like a copy of the actual config?)

**Review by dellisd — CHANGES_REQUESTED** [review]
The icons under `icons/vancouver` should be removed and put into the other PR with the other SkyTrain stuff.

For all of the VIA lines, I'd make each of their IDs `via/{name}`. This just helps group them together, and it'll also change how the icons are sorted in the station label. They're sorted alphabetically by ID so VIA would go last and I think that makes the most sense.

Also, if we could just keep the VIA lines all yellow for now. We can add colour later if we decide it makes more sense.

Lastly, there's still the `viarail.json` file which I assume is the original file with all of the data before it was split up. I don't think we'll need to keep that one.

**Comment by davidbellerive** [comment]
Changes have been implemented, copy of config file removed as it was misplaced.

File: data/viarail/montrealmaintenance.json [hunk 2]
```diff
@@ -0,0 +1,113 @@
+{
+"type": "FeatureCollection",
+"metadata": { "id": "montrealmaintenance", "type": "rail-yard", "color": "#FFCC00", "icon": "viarail\/viarail.svg", "name": "VIA Rail Montreal Maintenance Centre", "description": "Major maintenance facility for VIA Rail located in Montreal.", "sources": [ "VIA Rail Canada" ], "filterKey": "montrealmaintenance" },
```
Comment (dellisd) [inline data/viarail/montrealmaintenance.json:3]:
```suggestion
"metadata": { "id": "montrealmaintenance", "type": "rail-yard", "color": "#FFCC00", "icon": "viarail\/viarail.svg", "name": "VIA Rail Montreal Maintenance Centre", "description": "Major maintenance facility for VIA Rail located in Montreal.", "sources": [ "VIA Rail Canada" ], "filterKey": "via/corridor" },
```

File: data/viarail/vancouvermaintenance.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{
+"type": "FeatureCollection",
+"metadata": { "id": "vancouvermaintenance", "type": "rail-yard", "color": "#FFCC00", "icon": "viarail\/viarail.svg", "name": "VIA Rail Vancouver Maintenance Centre", "description": "Maintenance facility for VIA Rail located in Vancouver.", "sources": [ "VIA Rail Canada" ], "filterKey": "vancouvermaintenance" },
```
Comment (dellisd) [inline data/viarail/vancouvermaintenance.json:3]:
```suggestion
"metadata": { "id": "vancouvermaintenance", "type": "rail-yard", "color": "#FFCC00", "icon": "viarail\/viarail.svg", "name": "VIA Rail Vancouver Maintenance Centre", "description": "Maintenance facility for VIA Rail located in Vancouver.", "sources": [ "VIA Rail Canada" ], "filterKey": "via/canadian" },
```

File: data/viarail/torontomaintenance.json [hunk 4]
```diff
@@ -0,0 +1,160 @@
+{
+"type": "FeatureCollection",
+"metadata": { "id": "torontomaintenance", "type": "rail-yard", "color": "#FFCC00", "icon": "viarail\/viarail.svg", "name": "VIA Rail Toronto Maintenance Centre", "description": "Major maintenance facility for VIA Rail located in Toronto.", "sources": [ "VIA Rail Canada" ], "filterKey": "torontomaintenance" },
```
Comment (dellisd) [inline data/viarail/torontomaintenance.json:3]:
```suggestion
"metadata": { "id": "torontomaintenance", "type": "rail-yard", "color": "#FFCC00", "icon": "viarail\/viarail.svg", "name": "VIA Rail Toronto Maintenance Centre", "description": "Major maintenance facility for VIA Rail located in Toronto.", "sources": [ "VIA Rail Canada" ], "filterKey": "via/corridor" },
```

File: data/viarail/winnipegmaintenance.json [hunk 5]
```diff
@@ -0,0 +1,26 @@
+{
+"type": "FeatureCollection",
+"metadata": { "id": "winnipegmaintenance", "type": "rail-yard", "color": "#FFCC00", "icon": "viarail\/viarail.svg", "name": "VIA Rail Winnipeg Maintenance Centre", "description": "Maintenance facility for VIA Rail located in Winnipeg.", "sources": [ "VIA Rail Canada" ], "filterKey": "winnipegmaintenance" },
```
Comment (dellisd) [inline data/viarail/winnipegmaintenance.json:3]:
```suggestion
"metadata": { "id": "winnipegmaintenance", "type": "rail-yard", "color": "#FFCC00", "icon": "viarail\/viarail.svg", "name": "VIA Rail Winnipeg Maintenance Centre", "description": "Maintenance facility for VIA Rail located in Winnipeg.", "sources": [ "VIA Rail Canada" ], "filterKey": "via/canadian" },
```

**Review by dellisd — CHANGES_REQUESTED** [review]

**Pull request merged**